src/App.js

```diff
@@ -1,5 +1,133 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
 class App {
-  async play() {}
+  async play() {
+    Console.print('숫자 야구 게임을 시작합니다.');
+
+    while (true) {
+      await this.gameFlow();
+      const userRestartInput = await this.getUserRestartInput();
+      if (userRestartInput !== 1) {
+        break;
+      }
+    }
+  }
+  // 게임 실행 흐름을 책임지는 함수
+  async gameFlow() {
+    const COMPUTER_ANSWER = this.createComputerAnswer();
+    let isUserWon = false;
+
+    while (!isUserWon) {
+      isUserWon = await this.checkUserGuess(COMPUTER_ANSWER);
+    }
+    return;
+  }
+
+  // 사용자 입력을 받는 getUserInput
+  async getUserInput() {
+    try {
+      const answer = await Console.readLineAsync('숫자를 입력해주세요 : ');
+
+      const userInput = answer.split('');
+
+      if (
+        userInput.length !== 3 ||
+        userInput.some(char => isNaN(parseInt(char, 10))) ||
+        new Set(userInput).size !== 3
+      ) {
+        throw new Error(
+          '[ERROR] 1에서 9 사이의 서로 다른 세 숫자를 입력해 주세요.'
+        );
+      }
+
+      return userInput.map(char => parseInt(char, 10));
```

Comment:
오옹 저는 숫자로 변환해주는 부분에 Number 를 사용했는데, 
생각해보니 parseInt 가 맥락상으로 조금 더 적절하지 않았나 생각이 들어서 돌아보게 되네요..ㅎㅎ
`parseInt`의 경우에는 기본이 10진수 변환이기 때문에 두번째 인수인 10은 생략해주셔도 될 것 같아요!

```suggestion
      return userInput.map(char => parseInt(char));
```

Reply:
이부분 자세히 알아보니, 
에어비앤비 코드 컨벤션에 두번째 인수를 생략하지 말라는 가이드가 있군요 하핳..🙂
제가 오히려 배워갑니다 :)

Reply:
오 저도 이번에 에어비앤비 컨벤션을 보면서 처음 알게 된 부분이에요 😮

Reply:
저는 map(Number)로 작성해서 컨벤션을 지키지 못했었는데, 동훈님은 컨벤션에 따라 char => parseInt(char, 10)으로 작성하셔서  콜백의 parameter를 명시적으로 명명하신 부분이 좋다고 생각합니다 👍